aider/models.py

```diff
@@ -426,6 +426,18 @@ def validate_environment(self):
 
         return res
 
+def register_models(model_def_fnames):
+    for model_def_fname in model_def_fnames:
+        if not os.path.exists(model_def_fname):
+            continue
+        print(f"Registering model definition from {model_def_fname}")
```

Comment:
Maybe accumulate a string of these and return it so that it can be emitted via io.tool_output() in main?